src/deck/interface/deck_core.h

```diff
@@ -89,7 +77,17 @@ typedef struct deck_driver {
   uint8_t pid;
   char *name;
 
-  /* Periphreal and Gpio used _dirrectly_ by the driver */
+  /*
+   * Periphreal and Gpio used _dirrectly_ by the driver.
```

Comment:
maybe install a spell checker? (and be prepared for the shock when you browse existing code).

Reply:
Squiggly.